worker/fbs/notification.fbs

```diff
@@ -53,20 +53,20 @@ union Body {
   FBS.Producer.SendNotification,
   FBS.DataProducer.SendNotification,
   // Notifications from worker.
-  FBS.Transport.TraceNotification,
+  Transport_TraceNotification: FBS.Transport.TraceNotification,
```

Comment:
Why this?

Reply:
I mean: why just here and in a very few cases below?

Reply:
Due to [Same table names in different namespaces not supported](https://github.com/planus-org/planus/issues/180).

The solution is creating an alias which adds the namespace to the type.  I'll make a PR that creates an alias for all notification and request bodies, so we are consistent, which in fact results in shorter and more consistent names, taking `FBS` out of the type name.